lib/utils/logger.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file Sequelize module for debug and deprecation messages.
+ * It require a `context` for which messages will be printed.
+ *
+ * @module logging
+ * @access package
+ */
+import nodeDebug from 'debug';
+import util from 'util';
+
+/**
+ * The configuration for sequelize's logging interface.
+ *
+ * @access package
+ */
+export interface LoggerConfig {
+  /**
+   * The context which the logger should log in.
+   *
+   * @default 'sequelize'
+   */
+  context?: string;
```

Comment:
I suppose you removed the `debug` property because it isn't used anywhere in the codebase. @sdepold do you know why we had a `debug` property in the logger config in the first place?

Reply:
Nope no idea